saleor/product/utils.py

```diff
@@ -1,11 +1,15 @@
 from collections import namedtuple
-
```

Comment:
This line should stay there. Standardlib imports should be separated from project imports